cmd/argo/commands/watch.go

```diff
@@ -60,17 +66,20 @@ func watchWorkflow(wfName string) {
 		}
 		errors.CheckError(err)
 		wf := event.Object
-		printWorkflowStatus(wf)
+		if wf == nil {
```

Comment:
I'm pretty sure nil means error?

Reply:
Correct: if the `wf` does not exists we don't want to print it.